topics/EXP-output.rst

```diff
@@ -54,6 +58,11 @@ the Doxygen generated man pages or the source code itself.
   for any component but makes the most sense for near-equilibrium
   galaxy components such as disks, bulges and halos.
```

Comment:
Perhaps we should mention that `outchkpt` and `outchkptq` are for checkpointing, as the name suggests.  Each successive call will write a new checkpoint file with the same name, saving the previous file to `*..bak`